README.rst

```diff
@@ -42,8 +42,35 @@ The created venv should be located somewhere around .tox/dev/Scripts.
 
 Versioning
 ==========
-To track the versions of this python package we use `setuptools-scm <https://pypi.org/project/setuptools-scm/>`_.
-There are different kind of ways to achieve the version tracking. We will use the `pyproject.toml` usage.
+Short background information about versioning of python packages.
+At the moment (10.02.2021) there are `**seven** ways to define the version of your package <https://packaging.python.org/guides/single-sourcing-package-version/>`_.
```

Comment:
Das `seven` wird in der Vorschau nicht dick angezeigt. Scheinbar schafft er nicht dick und Link gleichzeitig oder es ist nicht die richtige Syntax.

Reply:
Dann nehmen wir das "dick" einfach raus.